### PR TITLE
K8s doc fixes

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- removed blank lines causing tables not to render on some pages
- removed component links for charms not specifically accessible in CharmHub (which also removes some breaking links)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

### CODE REVIEW

File: templates/kubernetes/docs/charm-docker-registry.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ juju run-action --wait docker-registry/0 start
 | <a id="table-storage-read-only"> </a> storage-read-only | boolean | False | Enable/disable the "readonly" storage maintenance option. False, the default, disables this option in the registry config file.  |
 | <a id="table-storage-swift-authurl"> </a> storage-swift-authurl | string |  | The URL of the keystone used to authenticate to swift.  |
 | <a id="table-storage-swift-container"> </a> storage-swift-container | string | docker-registry | The name of the swift container that will hold the images.  |
-| <a id="table-storage-swift-domain"> </a> storage-swift-domain | string |  | OpenStack Identity v3 API domain.  |
+| <a id="table-storage-swift-domain"> </a> storage-swift-domain | string |  | Openstack Identity v3 API domain.  |
```
Comment (anthonydillon) [inline templates/kubernetes/docs/charm-docker-registry.md:242]: Should be OpenStack, right?

File: templates/kubernetes/docs/charm-docker-registry.md [hunk 2]
```diff
@@ -547,7 +547,7 @@ juju config docker-registry \
 
 >Note: If any of the above config options are set, then they must all be set. Optional params are noted below.
 
-It is possible to configure the swift backend with an OpenStack domain name for Identity v3. To enable this, set the following optional config parameter:
+It is possible to configure the swift backend with an Openstack domain name for Identity v3. To enable this, set the following optional config parameter:
```
Comment (anthonydillon) [inline templates/kubernetes/docs/charm-docker-registry.md:550]: Same here

**Review by anthonydillon — APPROVED** [review]
LGTM with small comments

File: templates/kubernetes/docs/charm-docker-registry.md (hunk 1 above)
Comment (sowasred2012) [inline templates/kubernetes/docs/charm-docker-registry.md:242]:
```suggestion
| <a id="table-storage-swift-domain"> </a> storage-swift-domain | string |  | OpenStack Identity v3 API domain.  |
```

File: templates/kubernetes/docs/charm-docker-registry.md (hunk 2 above)
Comment (sowasred2012) [inline templates/kubernetes/docs/charm-docker-registry.md:550]:
```suggestion
It is possible to configure the swift backend with an OpenStack domain name for Identity v3. To enable this, set the following optional config parameter:
```

**Pull request merged**